pkg/sessions/persistence/manager.go

```diff
@@ -52,6 +57,10 @@ func (m *Manager) Save(rw http.ResponseWriter, req *http.Request, s *sessions.Se
 	return tckt.setCookie(rw, req, s)
 }
 
+// func (m *Manager) SaveUserSession(ctx context.Context, s *sessions.SessionState, value string, exp time.Duration) error {
+// 	return m.Store.SaveUserSession(ctx, s, value, exp)
+// }
```

Comment:
Can delete this commented-out stuff I think.

Reply:
done.